File: src/5장/5-4.py
def sum(A,B):
  return A+B
def sub(A,B):
  return A-B
def mul(A,B):
  return A*B
def div(A,B):
  return int(A/B)
def sur(A,B):
  return A%B

a,b= map(int, input().split())
print(sum(a,b))
print(sub(a,b))
print(mul(a,b))
print(div(a,b))
print(sur(a,b))

File: src/5장/5-4.java
import java.util.*;
import java.io.*;

public class Main {
	public static void main(String[] args) throws IOException {
        Scanner scan=new Scanner(System.in);
        int A,B;
        A=scan.nextInt();
        B=scan.nextInt();
		System.out.println(sum(A,B));
		System.out.println(sub(A,B));
		System.out.println(mul(A,B));
		System.out.println(div(A,B));
		System.out.println(sur(A,B));
	}
	static int sum(int a,int b){
		return a+b;
	}
	static int sub(int a,int b){
		return a-b;
	}
	static int mul(int a,int b){
		return a*b;
	}
	static int div(int a,int b){
		return a/b;
	}
	static int sur(int a,int b){
		return a%b;
	}
	
}